apps/frontend/src/modules/project-api-keys/project-api-keys-api.ts

```diff
@@ -35,6 +37,22 @@ class ProjectApiKeysApi extends BaseHTTPApi {
 
 		return await response.json<ProjectApiKeyCreateResponseDto>();
 	}
+
+	public async patch(
```

Comment:
We need to delete the previous key and create a new one, not to update it. We also will need to use existing post method for this, not patch.